ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/proxy/etcd-encryption/deprovision/cucushift-installer-rehearse-gcp-ipi-proxy-etcd-encryption-deprovision-chain.yaml

```diff
@@ -0,0 +1,12 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-proxy-workers-rhel8-etcd-encryption-deprovision
+  steps:
```

Comment:
And the following steps are wrong, which are specific for aws

Reply:
Yeah, fixed.